content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Creating a self-managed, multi-device multisig wallet"
+date: 2022-02-22T16:03:11.470Z
+lastmod: 2022-02-22T16:03:11.470Z
+weight: 6
+draft: false
+# search related keywords
+keywords: ["mac, macos, windows, linux, ios, android"]
+---
+
+{{< notice note >}}
+This tutorial is for someone that would like their transactions to be signed from multiple devices. In this workflow, we will be using a desktop, laptop, and a mobile phone to create a 2-of-3 multisig wallet. You can also use this workflow as a template to easily create another version of a multisig wallet.
```

Comment:
maybe:
You can also use this workflow as a template to create more complex multisig wallets.